### PR TITLE
Mdns

### PR DESCRIPTION
**Pull request by gferraro**
Added mdsnjava jar to try fix problems with dns lookup

### CODE REVIEW

File: app/src/main/java/nz/org/cacophony/sidekick/DeviceDiscovery.kt [hunk 1]
```diff
@@ -70,7 +74,13 @@ class DiscoveryManager(
...
             }
         }
-        if (!listenerFound) {
+
+        if (listenerFound) {
+            var localListener = listener;
+            if (localListener != null) {
+                localListener.updateConnected();
+            }
+        }else{
```
Comment (mjs) [inline app/src/main/java/nz/org/cacophony/sidekick/DeviceDiscovery.kt]: spaces

File: app/src/main/java/nz/org/cacophony/sidekick/DeviceDiscovery.kt [hunk 2]
```diff
@@ -119,6 +131,10 @@ class DeviceListener(
 ) : NsdManager.DiscoveryListener {
```
Comment (mjs) [inline app/src/main/java/nz/org/cacophony/sidekick/DeviceDiscovery.kt]: Is this still supposed to subclass NsdManager stuff with the new library? I would have thought there would be no need to involve NsdManager now.

**Comment by mjs** [comment]
Thanks for making this work. I'm very keen to see how it goes.

File: app/src/main/java/nz/org/cacophony/sidekick/DeviceDiscovery.kt (hunk 2 above)
Comment (gferraro) [reply app/src/main/java/nz/org/cacophony/sidekick/DeviceDiscovery.kt]: I think for now it's easier to use both, the stuff I added only checks for new devices on resume. If this fixes problems we can looking at only using one.

**Pull request merged**